src/workarounds.rs

```diff
@@ -0,0 +1,113 @@
+use bindings::{
+    Microsoft::UI::Xaml::*,
+    Windows::Win32::{
+        Foundation::{BOOL, HWND, RECT},
+        UI::{
+            HiDpi::GetDpiForWindow,
+            WindowsAndMessaging::{
+                GetSystemMetrics, GetWindowRect, SetWindowPos, SM_CXSCREEN, SM_CYSCREEN,
+                SWP_NOMOVE, SWP_NOSIZE,
+            },
+        },
+    },
+};
+use windows::{Interface, HRESULT};
+
+#[repr(transparent)]
+struct IWindowNative(windows::IUnknown);

```

Comment:
Where is `IWindowNative` defined? Would love to avoid having to define interfaces by hand. On the other hand, https://github.com/microsoft/windows-rs/issues/81 will eventually support.

Reply:
`microsoft.ui.xaml.window.h`; tough to say if this is a WinUI or Win32 Metadata _issue_ but looks like we got both bases covered now, thanks! https://github.com/microsoft/win32metadata/issues/538